rotkehlchen/tests/conftest.py

```diff
@@ -34,7 +34,7 @@
 SUBPROCESS_TIMEOUT = 30
 
 
-class TestEnvironment(SerializableEnumMixin):
+class EnvironmentForTests(SerializableEnumMixin):
```

Comment:
nitpick: Rather than doing a renaming...  you could just do this
```suggestion
class TestEnvironment(SerializableEnumMixin):
    __test__ = False
```

https://github.com/pytest-dev/pytest/pull/2099

Reply:
Didn't want to do it this was because declaring a `__test__` parameter in an enum looks strange to me, but okay.